app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
+    def __init__(self, nickname: str, favourite_dish: str) -> None:
+        super().__init__(nickname)
+        self.__favourite_dish = favourite_dish
```

Comment:
Methods should be `protected`, according to the task (with a `_`single underscore), instead of `__private` to allow subclass access and maintain flexibility in inheritance.
The rest of the code is good :)